### DISCUSSION TITLE
Confusion aroung netcommon releases

### DISCUSSION THREAD

**Discussion by Dalle55** [ansible-collections/ansible.netcommon#552] [General]
Hi, 

I am a bit confused about the releases available and also ansible support for the netcommon module.

On the main page of this module: it says latest release is 2.0.0 which is more than 2 years old. We are running version 3.1.1 in our installation. Where is the information on that release?

Also, the information states that the module is tested and supported on ansible ver. 2.9.10 - well, we are running ansible ver. 2.13.4

I am missing some updated information around this?

Can anyone help?

Thanks :-)

**Comment by NilashishC** [comment]
Hi @Dalle55. Could you please clarify where do you see "latest release as 2.0.0"?

> Also, the information states that the module is tested and supported on ansible ver. 2.9.10


It used to state `This collection has been tested against following Ansible versions: >=2.9.10.`
Now it says `This collection has been tested against following Ansible versions: >=2.14.0.`, since previous ansible-core versions are EoL now.

**Comment by Dalle55** [comment]
Hi Nilashish

Well this question is now almost 1 year old :-) But I saw it on the front page of the module, I can see now that it is changed.

I created the question when I was investigating an issue related to confirmed_commit. You can see the issue #551 here:
https://github.com/ansible-collections/ansible.netcommon/issues/551 

I am still waiting on help with that issue, nothing happens.

Maybe you can help?

Thanks
